Assorted map tweaks/fixes

atlantiscze opened the pull request:
- Fixes active edges at Vox skipjack and ERT Shuttle: Fixes #12782 - I actually believe the skipjack was fine anyway, and that the problem was only caused by ERT Shuttle, but i updated skipjack too, so all shuttles now have standardised thrusters with airless plating and rglass cover panel.
- (BONUS) Adds a debug code that may be enabled during compile time (#define ZASDBG) that prints areas with active edges to world <<. Obviously defaults to disabled. Should make it easier to trace down future problems with this.
- Remaps the abandoned shuttle near telecomms. Replaces shuttle turfs with normal walls/floors. Adds a VERY simple power system to it (slightly damaged so you have to patch it up). Most importantly, replaces the APC with disabled variant, to prevent alarms from showing up shortly after roundstart (this part is a bugfix).

Raptor1628 commented:
Pics of the new shuttle?

PsiOmegaDelta commented on code/ZAS/Controller.dm:
```diff
@@ -341,6 +341,13 @@ Total Unsimulated Turfs: [world.maxx*world.maxy*world.maxz - simulated_turf_coun
+		world << "ZASDBG: Active edge! Areas: [get_area(pick(ZE.A.contents))] / [get_area(pick(ZE.B.contents))]"
+	else
+		world << "ZASDBG: Active edge! Area: [get_area(pick(E.A.contents))]"
+	#endif
```
Or could simply use the admin_notice(string, R_DEBUG) method to always print it to everyone with debug access (same as initialize objects/areas, etc. does it),

atlantiscze replied:
I'm not sure about this. It is not exclusive for round-start active edges, it reports any edge that becomes active. It may spam the chat quite intensively during rounds where breaches, etc. occur a lot.

atlantiscze commented:
@Raptor1628 

This is how it looks from map editor perspective. Keep in minds the walls look very different in game:
http://i.imgur.com/nl52QlB.png

And this is an example screenshot from the game, for comparison:
http://i.imgur.com/1NaDG6Z.png

The layout itself is more or less how the shuttle looked before, i just replaced the walls/floors/etc, and added few cables and stuff. With some IC love and care it might make a nice antag hideout, assuming you can find it.

PsiOmegaDelta commented:
Well, I doubt the shuttle will ever be used as an antag hideout due to our stance on going space jumping looking for things but other than that
:+1:

Raptor1628 commented:
If someone ports Goon Pods I will personally change the space exploration rules. Looks good though.
